aviary/examples/run_reserve_mission_cruise_fixedrange.py

```diff
@@ -0,0 +1,155 @@
+"""
+This is a slightly more complex Aviary example of running a coupled aircraft design-mission optimization.
+It runs the same mission as the `run_basic_aviary_example.py` script, but it uses the AviaryProblem class to set up the problem.
+This exposes more options and flexibility to the user and uses the "Level 2" API within Aviary.
+
+We define a `phase_info` object, which tells Aviary how to model the mission.
+Here we have climb, cruise, and descent phases.
+We then call the correct methods in order to set up and run an Aviary optimization problem.
+This performs a coupled design-mission optimization and outputs the results from Aviary into the `reports` folder.
+"""
+import aviary.api as av
+
+
+phase_info = {
+    "pre_mission": {"include_takeoff": False, "optimize_mass": True},
+    "climb": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
```

Comment:
I wonder if it would be better to load in a base phase_info from our aircraft models instead of repeating all of the regular phases here. It is redundant, and doesn't highlight the actual changes that are needed to activate the reserve missions.

So, instead of this big dictionary, just import phase_info, add the 'cruise_reserve" phase, and edit the keys in post_mission.  That might be a bit clearer for the user about what they need to do to run a setup a reserve run. Also, it is more maintainable, since all of the normal flight phases are coming from 1 place.

Reply:
Done!